lib/widget/download/download_grid.dart

```diff
@@ -55,7 +55,6 @@ class _DownloadGridState extends State<DownloadGrid> {
                 height: resolveIconSize(fileType),
                 child: SvgPicture.asset(
                   FileUtil.resolveFileTypeIconPath(fileType.name),
-                  color: FileUtil.resolveFileTypeIconColor(fileType.name),
```

Comment:
Please roll this change back. Removing the color field would result in black icons which doesn't look good.

Reply:
i fix this issue